akka/src/main/scala/org/kaloz/gpio/dfgdfg/WebSocketActor.scala

```diff
@@ -0,0 +1,50 @@
+package org.kaloz.gpio.dfgdfg
+
+import akka.actor._
+import akka.http.scaladsl.model.ws._
+import akka.util.Timeout
+import com.typesafe.scalalogging.StrictLogging
+import org.json4s._
+import org.json4s.native.JsonMethods._
+import org.kaloz.gpio.User
+import org.kaloz.gpio.dfgdfg.WebSocketActor.{RegistrationClosed, Initialize, RegistrationOpened}
+import org.kaloz.gpio.dfgdfg.WebSocketMessageFactory._
+
+import scala.concurrent.duration._
+
+class WebSocketActor extends Actor with StrictLogging {
```

Comment:
There is no point in using StricLogging on an actor as there is a build in ActorLogging trait which gives you log object as well.